.ansible-lint.yml

```diff
@@ -0,0 +1,34 @@
+exclude_paths:
+  - .github
+  - .tox
+
+# Failed: 1348 failure(s), 97 warning(s) on 4230 files.
+skip_list:
+  - 'command-instead-of-module'
+  - 'command-instead-of-shell'
+  - 'deprecated-local-action'
+  - 'key-order[task]'
+  - 'jinja[invalid]'
+  - 'jinja[spacing]'
+  - 'schema[tasks]'
+  - 'name[missing]'
+  - 'name[play]'
+  - 'var-naming[no-reserved]'
+  - 'var-naming[pattern]'
+  - 'yaml[document-start]'
+  - 'yaml[empty-lines]'
+  - 'name[template]'
+  - 'name[casing]'
+  - 'risky-file-permissions'
+  - 'risky-shell-pipe'
+  - 'galaxy[no-changelog]'
+  - 'galaxy[version-incorrect]'
+  - 'ignore-errors'
```

Comment:
For actual playbooks this is probably a good type of rule, but I'm guessing it's hitting this in the awx.awx collection tests. You can't test effectively without a lot of error scenarios that shouldn't happen if you're following best practices.

I'd like to know how many of these rules are skipped due to the collection tests, and for maintainability, I wonder if it would be better to have a separate list for those. I'm talking about `awx_collection/tests/integration`